ui/src/pages/user_account_page/UserAccountPage.tsx

```diff
@@ -11,19 +10,30 @@ import { Disable2FAModal } from './Disable2FAModal'
 import { DeleteAccountModal } from './DeleteAccountModal'
 import { createModalState } from '../../components/Modal'
 import { Table, TableRow } from '../../components/Table'
-import { Button } from '../../components/Button'
+import { Button, IconButton } from '../../components/Button'
+import { Tooltip } from '../../components/Tooltip'
 
 export function UserAccountPage(): JSXElement {
   const { t } = useLocale()
   const { user } = useUser()
 
+  const [sending, setSending] = createSignal(false)
+
+  const onResendVerificationMail = async () => {
+    setSending(true)
+    await resendVerificationMail()
+    setSending(false)
+  }
+
   const [modalState, openModal, closeModal] = createModalState(
     'password',
     'enable2FA',
     'disable2FA',
     'deleteAcount'
   )
 
+  const twoFactorEnabled = () => user().twoFactorEnabled
```

Comment:
Why make this a method?

Reply:
Its a [derived signal](https://docs.solidjs.com/concepts/derived-values/derived-signals), I thought it would be more readable this way.